cv-basics/blob detection.ipynb

```diff
@@ -0,0 +1,176 @@
+{
```

Comment:
<p>Add table of every OpenCV function used with their descriptions and parameters, like <a href="https://github.com/SRA-VJTI/Intro_to_CV/blob/main/python-numpy-basics/Numpy_Part_1.ipynb" target="_blank">this</a>. </p><p>Add the link to the doc of OpenCV</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>